internal/logger/logger.go

```diff
@@ -155,6 +156,7 @@ func Errorf(format string, v ...interface{}) {
 // Fatal prints an error log and exits with non-zero exit code.
 func Fatal(format string, v ...interface{}) {
```

Comment:
This function is currently not unit tested, and am not sure how to go about it as deals with panic -> log -> exit.

Reply:
Captured in a google internal cleanup ticket.